tests/query/parser/test_expressions.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from snuba.query.expressions import (
+    Column,
+    CurriedFunctionCall,
+    FunctionCall,
+    Literal,
+)
+from snuba.query.parser.expressions import parse_aggregation
+
+test_data = [
+    (
+        ["count", "event_id", None],
+        FunctionCall(None, "count", (Column(None, "event_id", None),)),
+    ),  # Simple aggregation
+    (
+        ["count()", "", None],
+        FunctionCall(None, "count", ()),
+    ),  # Common way to provide count()
+    (
+        ["uniq", "platform", "uniq_platforms"],
+        FunctionCall("uniq_platforms", "uniq", (Column(None, "platform", None),)),
+    ),  # Use the columns provided as parameters
+    (
+        ["topK(1)", "platform", "top_platforms"],
+        CurriedFunctionCall(
+            "top_platforms",
+            FunctionCall(None, "topK", (Literal(None, 1),)),
+            (Column(None, "platform", None),),
+        ),
+    ),  # Curried function
+    (
+        ["apdex(duration, 300)", "", "apdex_score"],
```

Comment:
Sentry is also passing `quantile(0.95)(duration)` for the pxx field aliases, at least for a while longer.

Reply:
meh ..... I guess this way it would be interpreted as a column and in the end escaped thus probably won't work. 
Will try. The right way for this is using apdex as a snuba function, with snuba syntax. I can go and fix it around as soon as I will have built the proper query processor.

Reply:
If snuba exposed a `pxx(duration, 0.95)` function similar to apdex() that would be fantastic. I'm happy to help make that happen.

Reply:
now we parse the full expression. About exposing that function, sure we can add a queruy processor for that.